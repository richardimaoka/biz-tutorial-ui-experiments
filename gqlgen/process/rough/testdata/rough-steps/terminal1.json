{
  "phase": "Installから動かす直前まで",
  "instruction": "mkdir gqlgen-todos",
  "commit": "",
  "comment": ""
}
