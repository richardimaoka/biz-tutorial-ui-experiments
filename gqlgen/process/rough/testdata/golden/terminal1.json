[
  {
    "fromRoughStep": true,
    "subId": "terminalCommandStep",
    "step": "",
    "focusColumn": "Terminal",
    "terminalText": "mkdir gqlgen-todos",
    "terminalType": "command"
  }
]